src/encoded/static/components/app.js

```diff
@@ -188,6 +190,30 @@ class Timeout {
 }
 
 
+const EulaModal = ({ closeModal, signup }) => { return (
+    <Modal>
+        <ModalHeader title="Creating a new account" closeModal={closeModal} />
+        <ModalBody>
+            <div>
+                <p>
+                    You are about to create an account in ENCODED. Please, have a look at out <a href="https://www.stanford.edu/site/terms/">terms of service</a> and our <a href="https://www.stanford.edu/site/privacy/">privacy policy</a>.
+                </p>
+            </div>
+        </ModalBody>
+        <ModalFooter
+            closeModal={closeModal}
+            submitBtn={signup}
+            submitTitle="Proceed"
```

Comment:
"Proceed" or "Continue"?

Reply:
`account in ENCODED` Maybe `You are about to create an ENCODE account.` ENCODED isn't a thing.

Reply:
Typo: `Please have a look at out`

Maybe `Please have a look at the terms of service and privacy policy.`

Reply:
I meant "our" instead of "out". I changed the text to reflect the idea you have.